src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -3054,6 +3160,27 @@ public void EnumMemberAttributeUpdate()
                 Diagnostic(RudeEditKind.ChangingAttributesNotSupportedByRuntime, "[A2]X", FeaturesResources.enum_value));
         }
 
+        [Fact]
+        public void Enum_Member_Attribute_InsertDelete()
```

Comment:
Either this test is named wrong, of the Obsolete attribute is missing in `srcB1`

Reply:
Should be Enum_Member_Attribute_InsertDeleteAndUpdate